### ISSUE TITLE
Android - Search Results Screen - Duplicate Links for the Text "Continue Reading."

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#26]
There is an issue when TalkBack is reading the text/link ”Continue reading.” because a screen reader user will wonder why there are duplicate links. The user will not know what the continue to read is about. These items should have some sort of accessible/off-screen text that mentions which item it is reading.

An example would be for a search for the word “test” as follows: “Continue reading about Grievance Mechanisms” or “Grievance Mechanisms continue reading”. Where the words “about Grievance Mechanisms” or “Grievance Mechanisms” would be off-screen

Please see attached screenshot.

![android - search results screen - duplicate link for continue reading](https://user-images.githubusercontent.com/10423487/57150622-c8a39400-6d9c-11e9-9627-52380322f13e.png)

**Comment by KyleLivingstonDOL** [comment]
It looks like this is done. UAT will begin June 10.

**Comment by jfc3-dol** [comment]
Closing this issue because TalkBack now is reading the screen name after the text "continue Reading" as expected.
